unifi/account.generated.go

```diff
@@ -23,7 +23,7 @@ type Account struct {
 	NoDelete bool   `json:"attr_no_delete,omitempty"`
 	NoEdit   bool   `json:"attr_no_edit,omitempty"`
 
-	IP               string `json:"ip"`                           // ^(([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])$|^$
```

Comment:
You can just add a hard coded escape in the generator here: https://github.com/paultyng/go-unifi/blob/ae40573bb7b42c16c05fa655d640836f3c4ccfb2/fields/main.go#L195-L207

So that at least it will come through this way next time. But I think I should generally just remove `omitempty` if I observe the `|^$` in the regex, I'll see if I can make that change as well and then we could remove the special casing.

Reply:
Oh sorry I misread, you are adding the `omitempty`, not removing in this case.